### PR TITLE
[Commands] Add #findclass [search criteria] command.

### PR DESCRIPTION
**Pull request by KinglyKrab**
- Allows GMs to find a class by name or ID.
- Modify some verbiage in command messages that were improper.

### CODE REVIEW

**Review by fryguy503 — APPROVED** [review]
LGTM

File: zone/command.cpp [hunk 1]
```diff
@@ -2688,10 +2689,51 @@ void command_showskills(Client *c, const Seperator *sep)
...
+		int search_id = atoi(sep->argplus[1]);
+		std::string class_name = GetClassIDName(search_id);
+		if (class_name != std::string("")) {
+			c->Message(Chat::White, "Class %d: %s", search_id, class_name.c_str());
+			return;
+		}
+	} else {
+		const char *search_criteria = sep->argplus[1];
+		int found_count = 0;
+		char class_name[64];
```
Comment (Akkadius) [inline zone/command.cpp:2706]: There's an opportunity to simplify this to use some of the CPP string functionality versus the old C character array API's

See `command_list` for example where string search is using the `std::string` object which has built in functions like `find`

```cpp
/**
 * Filter by name
 */
if (search_string.length() > 0 && entity_name.find(search_string) == std::string::npos) {
	continue;
}
```

**Comment by Akkadius** [comment]
Left minor comments; LGTM otherwise

File: zone/command.cpp (hunk 1 above)
Comment (KinglyKrab) [reply zone/command.cpp:2706]: I'll change it when I get home. It was just a copy and paste of #findrace with minor alterations. I can change both for cleanup.

**Review by Akkadius — APPROVED** [review]

**Pull request merged**